equirectangular_topic_name: /equirectangular/image_raw
reference_save_dir: theta_s_references
